src/resources/js/accordion.js

```diff
@@ -0,0 +1,52 @@
+(function( window, $ ) {
+	var accordions = document.getElementsByClassName( 'accordion' );
+
+	var accordionIndex = ( undefined === accordionIndex ) ? 0 : accordionIndex++;
```

Comment:
In recent times we've mostly used `snake_case` even in our JS, though I do see the WP JS standards currently recommend `camelCase` (did that change sometime recently?) ... @borkweb you have any thoughts - should we generally use snakeCase in JS from now on?